### ISSUE TITLE
RC101

### ISSUE THREAD

**Issue by gpreising** [FlukeAndFeather/maxspeed#3]
1. Prerequisite versions: R 4.1.1, rrtools 0.1.5, tidyverse 1.3.1, devtools 2.4.2
2. A research compendium is collection of information related to an experiment that can be used to efficiently and accurately replicate said experiment. This includes things like file storage design, code comments, and anything else (like readme files) to facilitate reproducibility.
3. Yes, in my case they were posted on github. Other scientists trying to replicate the analysis may struggle with figuring out the right versions for the scripts posted as well as sifting through extraneous chunks of code that produced figures we didn't use for the publication.

**Comment by FlukeAndFeather** [comment]
> extraneous chunks of code that produced figures we didn't use for the publication

Haha yup, I've been there too! If you look at the R scripts in many papers' supplementals you'll find a lot calls to functions like `summary()` or other diagnostic tools, like figures that didn't make it into the publication. I think one could argue those are helpful because they guide the reader through exploration, but you could also argue they're distractions from the crucial material. What do you think?

**Comment by elhazen** [comment]
Do you have a link to the github for your paper? 

I also think that having a lot of code that was used in the process but may not have been important for the final paper could be a useful addition but I have usually have a 'sandbox' private repo and then make a nice clean public repo for publication. I wonder if it would be good to include the "lost" code moving forward?

**Comment by gpreising** [comment]
Yeah it's [here](https://github.com/gpreising/cnv_project_code). I agree with @FlukeAndFeather that they are a bit distracting. I think we reached a point where the paper just needed to get published after struggling with it for a while, so we just added the final code to the 'sandbox' repo that @elhazen mentioned and called it a day. Probably not the cleanest way to publish code, which is why I'm enrolled in this class :)
